securedrop/journalist_app/forms.py

```diff
@@ -37,10 +37,16 @@ def name_length_validation(form, field):
             .format(max_chars=Journalist.MAX_NAME_LEN)))
 
 
+def check_invalid_usernames(form, field):
+    if field.data in Journalist.INVALID_USERNAMES:
+        raise ValidationError(gettext(
+            "Invalid username"))
```

Comment:
Or maybe a text saying "This username is not allowed in SecureDrop", @emkll @eloquence some intput here will be useful.

Reply:
Given the 0.001% likelihood that any legit user is every going to see this message I'm not too fussed, but I'd suggest going with:
> This username is invalid because it is reserved for internal use by the software.

Reply:
Should I make this change?

Reply:
Yes please, thank you :)